resources/sqlite_logger.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import sqlite3
+import logging
+import time
+
+
+INIT_SQL = """CREATE TABLE IF NOT EXISTS log(
+                    TimeStamp TEXT,
+                    Source TEXT,
+                    LogLevel INT,
+                    LogLevelName TEXT,
+                    Message TEXT,
+                    Args TEXT,
+                    Module TEXT,
+                    FuncName TEXT,
+                    LineNo INT,
+                    Exception TEXT,
+                    Process INT,
+                    Thread TEXT,
+                    ThreadName TEXT
+               )"""
+
+INSERT_SQL = """INSERT INTO log(
+                    TimeStamp,
+                    Source,
+                    LogLevel,
+                    LogLevelName,
+                    Message,
+                    Args,
+                    Module,
+                    FuncName,
+                    LineNo,
+                    Exception,
+                    Process,
+                    Thread,
+                    ThreadName
+               )
+               VALUES (
+                    '%(dbtime)s',
+                    '%(name)s',
+                    %(levelno)d,
+                    '%(levelname)s',
+                    '%(msg)s',
+                    '%(args)s',
+                    '%(module)s',
+                    '%(funcName)s',
+                    %(lineno)d,
+                    '%(exc_text)s',
+                    %(process)d,
+                    '%(thread)s',
+                    '%(threadName)s'
+               );
+               """
+
+def getLogLevel():
+            f = open("log-level.txt", "r")
```

Comment:
Will throw error if file does not exist, please have a check or change mode to automatically create the file if it does not exist